index.html

```diff
@@ -7,33 +7,241 @@
+    <div class="section-sub">
+      <h1 class="title dark-blue">About</h1>
+
+      <div class="contents">
+        <p class="paragraph dark-blue">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button class="collab-button bg-dark-blue white">
+          <div class="cv-download">
+            <p>Get my CV</p>
+            <img class="icon" src="images/ic_download.png" alt="This is a download icon" width="24" height="18">
+          </div>
+        </button>
+      </div>
+
+      <div class="box-wrapper">
+        <ul class="box-container bg-bluish">
+          <li class="title-sub white">Languages</li>
+          <ul class="list-languages">
+            <li class="white">JavaScript(ES5/ES6)</li>
+            <li class="white">Ruby</li>
+            <li class="white">HTML</li>
+            <li class="white">CSS</li>
+          </ul>
+          <div class="pound css-background-img"></div>
+        </ul>
+        <ul class="box-container bg-redish">
+          <li class="title-sub white">Frameworks</li>
+          <ul class="list-frameworks">
+            <li class="white">React.js</li>
+            <li class="white">Ruby on rails</li>
+            <li class="white">RSpec</li>
+            <li class="white">Capibara</li>
+          </ul>
+          <div class="card css-background-img"></div>
+        </ul>
+        <ul class="box-container bg-yellowish">
+          <li class="title-sub white">Skills</li>
+          <ul class="list-skills">
+            <li class="white">Database Management</li>
+            <li class="white">CLI</li>
+            <li class="white">API Design</li>
+            <li class="white">Version Control</li>
+            <li class="white">Web Development</li>
+          </ul>
+          <div class="idea css-background-img"></div>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <!--Get Started Section-->
+  <div class="bg-offwhite">
+    <div class="top-edge-container">
+      <div class="geo-background-img"></div>
+    </div>
+    <section class="section-container">
+      <div class="section-sub">
+        <h1 class="title dark-blue">Get started</h1>
+        <div class="contents">
+          <p class="paragraph dark-blue">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </p>
+        </div>
+        <p class="small-title offwhite">Your name</p>
+        <div>
+          <div class="input-button dark-blue bg-white">Anastasiia</div>
+        </div>
+        <p class="small-title offwhite">Your email</p>
+        <div class="input-button bg-white dark-blue">anastasiia@gm...</div>
+        <p class="small-title offwhite">Your message</p>
+        <div class="bigger-box bg-white"></div>
+        <button class="collab-button bg-dark-blue white">Start collaboration</button>
+      </div>
```

Comment:
Great job on your contact section! I have a few suggestions that could make it look even better and closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=23-9&t=DOmVpbk2buQv96CX-0):

- [ ] Please check the top space between your `paragraph` and your `title`, as it may need some adjustment for better alignment.
- [ ] The correct `color` for the name of your inputs should be `#7D7C84` according to the[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=23-9&t=DOmVpbk2buQv96CX-0).
- [ ] Please align your `Start Collaboration` button to ensure that **it's centered and properly positioned within its container.**

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236647776-b0c4c551-7b5e-43ba-b741-9fe9d89ed361.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236647817-bb9a68c4-67b3-4e49-8be0-8db17f05ddc2.png)